### DISCUSSION TITLE
Support of older versions of GO

### DISCUSSION THREAD

**Discussion by minchopm** [snakers4/silero-vad#394] [Ideas]
Is it possible to add support of GO version >= 1.18, because of this requirement https://github.com/quic-go/quic-go/wiki/quic-go-and-Go-versions

**Comment by snakers4** [comment]
Hi @minchopm ,

We have a community example for GO provided by @streamer45 
I guess you can discuss his example here

**Reply by minchopm** [reply]
I have forked the @streamer45 example and downgraded the library to use older versions of GO [https://github.com/minchopm/silero-vad-go](https://github.com/minchopm/silero-vad-go), there were few problems - support of Mac M1 chipset require usage of longlong instead of long and usage of logslog lib that require 1.21
